tox_encryptsave/src/lib.rs

```diff
@@ -423,21 +406,17 @@ fn pass_key_new_test() {
     let passwd = [42; 123];
     let pk = PassKey::from_passphrase(&passwd).expect("Failed to unwrap PassKey!");
 
-    assert_ne!(pk.salt.0.as_ref(), &passwd as &[u8]);
-    assert_ne!(pk.salt.0.as_ref(), [0; SALT_LENGTH].as_ref());
-    assert_ne!(pk.key.0.as_ref(), &passwd as &[u8]);
-    assert_ne!(pk.key.0, [0; KEY_LENGTH]);
```

Comment:
How about pk.key tests?

Reply:
Same as with refreshing onion key tests - we can't access key from XSalsa20Poly1305 struct.